engine/src/util.cpp

```diff
@@ -2916,6 +2916,212 @@ MCU_path_has_extension(MCStringRef p_path)
                                     t_ext);
 }
 
+static void
+MCU_path_compute_split_unix(MCStringRef p_path,
+                            uindex_t p_split_at,
+                            uindex_t& x_dir_end,
+                            uindex_t& x_base_start)
+
+{
+    if (MCStringGetCharAtIndex(p_path, 0) == '/')
+    {
+        if (p_split_at == 0)
+        {
+            /* Make sure dir is / and base is everything after */
+            x_dir_end = 1;
+        }
+    }
+    else
+    {
+        if (p_split_at == 0)
+        {
+            x_dir_end = 0;
+            x_base_start = 0;
+        }
+    }
+}
+
+static void
+MCU_path_compute_split_win32(MCStringRef p_path,
+                             uindex_t p_split_at,
+                             uindex_t& x_dir_end,
+                             uindex_t& x_base_start)
```

Comment:
I don't think the outputs ever get read from in this function, so I suggest naming them `r_dir_end` / `r_base_start` according to our naming policy.